packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -102,7 +105,12 @@ const CFDsListing = () => {
                         <Text size='sm' line_height='m' weight='bold' color='prominent'>
                             {localize('CFDs')}
                         </Text>
-                        <div className='cfd-accounts__compare-table-title' onClick={toggleCompareAccountsModal}>
+                        <div
+                            className='cfd-accounts__compare-table-title'
+                            onClick={() => {
+                                history.push(routes.compare_cfds);
+                            }}
+                        >
```

Comment:
This looks awfully similar to the change below. The only difference is in mobile you have this additional prop `styles={{ marginLeft: '1rem' }}` Cant they be merged to a single component?

And i know this is unrelated to your change but
`<Localize i18n_default_text={accounts_sub_text} />`

this wont work 😢

Reply:
Do we need to create a card for the Localize part which might cause issues ?